### PR TITLE
Turn echo off  partially

### PR DESCRIPTION
**Pull request by basoundr**
Also replace the fixed path to VS Cmd prompt to Environment variable

Tagging @dotnet/roslyn-analysis  @srivatsn @mavasani @heejaechang  for review

### CODE REVIEW

File: cibuild.cmd [hunk 1]
```diff
@@ -12,9 +12,11 @@ if /I "%1" == "/release" set BuildConfiguration=Release&&shift&& goto :ParseArgu
 call :Usage && exit /b 1
 :DoneParsing
 
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
+call "%VS140COMNTOOLS%VsDevCmd.bat"
 
 msbuild /v:m /m %AnalyzersRoot%\BuildAndTest.proj /p:CIBuild=true /p:Configuration=%BuildConfiguration%
+
+@echo off
```
Comment (srivatsn) [inline cibuild.cmd]: Why just partially - why not the entire file?

Comment (basoundr) [inline cibuild.cmd]: I thought it might affect the results of the build but now it wouldn't. Will Turn 
echo off for the whole file

**Comment by basoundr** [comment]
@dotnet/roslyn-analysis @srivatsn @mavasani @heejaechang ping?

**Comment by srivatsn** [comment]
👍

**Comment by davkean** [comment]
:+1:

**Pull request merged**